tools/autograd/load_derivatives.py

```diff
@@ -85,11 +87,135 @@ def create_derivative(f: NativeFunction, formula: str, var_names: Tuple[str, ...
 
     return Derivative(
         formula=formula,
+        original_formula=original_formula,
         var_names=var_names,
         saved_inputs=saved_inputs,
         saved_outputs=saved_outputs,
     )
 
+
+def find_required_inputs_fw_grads(formula: str) -> Tuple[str, ...]:
```

Comment:
Not sure if mypy-strict is integrated into CI. I ran it locally on this PR and got some errors:
```
mypy --config mypy-strict.ini

tools/autograd/load_derivatives.py:99:12: error: Incompatible return value type (got "List[Any]", expected "Tuple[str, ...]")
tools/autograd/load_derivatives.py:101:1: error: Function is missing a return type annotation
tools/autograd/load_derivatives.py:124:18: error: Argument "var_type" to "ForwardDerivative" has incompatible type "Type"; expected "str"
tools/autograd/load_derivatives.py:126:32: error: Argument "required_inputs_primal" to "ForwardDerivative" has incompatible type "None"; expected "Tuple[str, ...]"
...
```

Reply:
Yes it is. I am fixing them now as I forgot to run it locally :/ 
I'll double check this one as well.